tests/epyccel/test_base.py

```diff
@@ -5,6 +5,13 @@
 from modules import base
 from conftest       import *
 
+#To delete after solve issue (is and is not not working in c) 
+def tmp_compare_epyccel(language, f, *args):
```

Comment:
You do not need to have 2 `compare_epyccel` functions. You can give the same result either by providing a default value for language (which will then have to be called using the keyword):
```
def compare_epyccel(f, *args, language='fortran'):
```
or by marking the failing tests as xfail:
```
@pytest.mark.parametrize("language",['fortran',pytest.param('c', marks = pytest.mark.xfail(reason="Issue 409: 'is'/'is not' not implemented in c")
def test_is_false(language):
    compare_epyccel(base.is_false, True, language=language)
    compare_epyccel(base.is_false, False, language=language)
```